l10n_eu_nace/models/res_partner_nace.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2019 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models, _
+
+
+class ResPartnerNace(models.Model):
+
+    _name = 'res.partner.nace'
+    _description = 'European NACE partner categories'
+    _order = 'code'
+    _parent_store = True
+
+    name = fields.Char()
+    parent_id = fields.Many2one(comodel_name="res.partner.nace")
+    code = fields.Char()
+    child_ids = fields.One2many(
+        'res.partner.nace', 'parent_id', string='NACE subcategories'
+    )
+    active = fields.Boolean(default=True)
```

Comment:
This is used by default to filter:

```suggestion
    active = fields.Boolean(index=True, default=True)
```